### DISCUSSION TITLE
input number "max" doesn't work

### DISCUSSION THREAD

**Discussion by lucapu88** [bootstrap-vue/bootstrap-vue#6185] [Q&A] [answered]
Hi I have a problem. I have this input where the property `max = "100"` doesn't work for me and then if I type something like: 12345678910 it makes me type it.

```
<b-form-input
          type="number"
          class="input-form-number-persone"
          v-model="number"
          placeholder="N° of people"
          inline
          min="1"
          max="100"
          @input="getPeople(number)"
        >
</b-form-input>
```

I also tried the modification within the method getPeople(), but it doesn't work:

```
    getPeople(text) {
      const numberPeople = this.number;
      if (numberPeople.length >= 3) {
        numberPeople.slice(0, 3); 
      }
      //this.saveEventPeople(text);
    },
```

**Comment by Hiws** [comment, marked as answer]
Both the `min` and `max` prop, are simply passed along to the native input attribute.
<https://www.w3schools.com/tags/att_input_max.asp>

These are used by the browser for validation when submitting a form, and doesn't stop the user from typing number higher or lower. (though, this can vary depending on the browser).

Try typing a number less than 1 or higher than `5` in the codepen, and submit the form.
The browser should give you a warning that the input has to be between 1 and 5.
<https://codepen.io/Hiws/pen/GRjrezo>

**Comment by lucapu88** [comment]
ok but I would like to ensure that the user cannot type a digit with a length not exceeding 3 digits ... with other poorer libraries it is feasible, with bootstrap vue no?

**Comment by Hiws** [comment]
You could write a `@keydown` function to prevent the keystroke if it exceeds the max.
